.github/workflows/evaluate-coderabbit-review.yml

```
@@ -3,6 +3,12 @@ name: Evaluate CodeRabbit Review
 on:
   pull_request_review:
     types: [submitted]
+  pull_request_review_comment:
+    types: [created, resolved]
```

actionlint (1.7.7)

invalid activity type "resolved" for "pull_request_review_comment" Webhook event. available types are "created", "deleted", "edited"

(events)